app/models/detailed_guide.rb

```diff
@@ -19,8 +19,13 @@ class DetailedGuide < Edition
   include Edition::RelatedPolicies
   include Edition::RelatedDocuments
 
-  validate :related_mainstream_content_valid?
-  validate :additional_related_mainstream_content_valid?
+  has_many :related_mainstream, foreign_key: "edition_id", dependent: :destroy
+
+  validate :related_mainstream_found
+
+  after_save :persist_content_ids
+
+  attr_reader :content_ids, :related_mainstream_content_ids
```

Comment:
Indeed! And I was only using `content_ids` in the tests, which isn’t a good reason. Fixed in 9291df5.